README.md

```diff
@@ -32,6 +32,9 @@ composer require yiisoft/request-model
 
 ## General usage
 
+According to [`yiisoft/middleware-dispatcher`](https://github.com/yiisoft/middleware-dispatcher) docs, you need to tweak 
```

Comment:
Can't get what is mean here :(

Reply:
May be instead of this text add information about configuration to "Installation" section.